Implement feed list using GtkStackSidebar

vrutkovs opened the pull request:
Final step for initial UI

Closes #1

Commit:
Implement feed list using GtkStackSidebar

Closes #1

ignatenkobrain commented:
From first glance looks good. I will test it ASAP.

What do you think about GtkScrollWindow as primary child for all stacks?

vrutkovs commented:
>What do you think about GtkScrollWindow as primary child for all stacks?

That ain't gonna work AFAIK, GtkStackSidebar creates its own GtkScrollWindow and those contradict.


The biggest minus in this implementation - its rather hard to enable selection mode for GtkStackSidebar, not sure if implementing feed removal would be simple

ignatenkobrain commented:
```python
$ jhbuild run gnome-news
18:41:53 ERROR	We should not get this type from sparql. name: title, type: <enum TRACKER_SPARQL_VALUE_TYPE_UNBOUND of type TrackerSparqlValueType>
Traceback (most recent call last):
  File "/home/brain/jhbuild/install/lib/python3.4/site-packages/gnomenews/window.py", line 105, in view_changed
    visible_view.update()
  File "/home/brain/jhbuild/install/lib/python3.4/site-packages/gnomenews/view.py", line 198, in update
    self.update_feeds()
  File "/home/brain/jhbuild/install/lib/python3.4/site-packages/gnomenews/view.py", line 152, in update_feeds
    self._add_new_feed(new_feed)
  File "/home/brain/jhbuild/install/lib/python3.4/site-packages/gnomenews/view.py", line 115, in _add_new_feed
    self.feed_stack.add_titled(flowbox, feed['url'], feed['title'])
TypeError: Argument 3 does not allow None as a value
```

ignatenkobrain commented:
if i will open feeds tab when feed added, but title is not ready

vrutkovs commented:
AFAIK we add the feed only after its fully loaded (w/ title), so this seems to be a libgrss problem

ignatenkobrain commented:
nope, when i switch to feeds tab it requests tracker to get channels , but actually channel in tracker dont have title yet, it will get in few seconds/minutes (depend on inet speed)

Commit:
Cleanup unused lines of code and run '_handle_insert_event' via idle_add

This would run the signal-emitting code last, so all titles and urls
will be fetched

Commit:
view: replace empty title with 'Unknown feed'

Commit:
tracker: don't load posts twice after post was marked as read

Commit:
Hide add button on post view

Commit:
logs: use tab to separate location and message

vrutkovs commented:
>channel in tracker dont have title yet

Wouldn't this be a part of ```GraphUpdated``` signal? Or it gets spawned several times?

ignatenkobrain commented:
yep, it is part of GraphUpdated signal. but now we don't handle it for channels. I.e. if channel adds in GraphUpdated I don't see any new channels in feeds tab.
